app/lib/aether_utils.coffee

```diff
@@ -64,12 +64,12 @@ functionParameters =
 
 # TODO Webpack: test to make sure this refactor works everywhere it's used
 module.exports.generateSpellsObject = (options) ->
-  {level, levelSession} = options
+  {level, levelSession, token} = options
   {createAetherOptions} = require 'lib/aether_utils'
   aetherOptions = createAetherOptions functionName: 'plan', codeLanguage: levelSession.get('codeLanguage'), skipProtectAPI: options.level?.isType('game-dev')
   spellThang = thang: {id: 'Hero Placeholder'}, aether: new Aether aetherOptions
   spells = "hero-placeholder/plan": thang: spellThang, name: 'plan'
-  source = levelSession.get('code')?['hero-placeholder']?.plan ? ''
+  source = token or levelSession.get('code')?['hero-placeholder']?.plan ? ''
```

Comment:
Can we add some short docs on what token is and why we have this shortcut?  Having trouble following this because I'm so unfamiliar with this part of the code base.

Reply:
Added docs to slab